packages/orbit-components/src/Timeline/Timeline.stories.tsx

```diff
@@ -116,107 +114,49 @@ export const WithWarning = ({ label, time, type, children }) => {
         We’ll forward you all refunds from the carrier(s) after we receive it.
       </TimelineStep>
     </Timeline>
-  );
-};
+  ),
 
-WithWarning.story = {
-  name: "with warning",
-  parameters: { info: "This is example of Timeline with warning type" },
-};
-
-WithWarning.args = {
-  label: "Action required",
-  time: "5th May 15:03",
-  type: Types.Success,
-  children: "The carrier has sent us a refund. There might be more depending on their policy",
-};
-
-WithWarning.argTypes = {
-  type: {
-    options: Object.values(Types),
-    control: {
-      type: "select",
-    },
+  parameters: {
+    info: "You can try all possible configurations of this component. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
-
-export const WithCritical = ({ label, time, type, children }) => {
-  return (
-    <Timeline dataTest={dataTest}>
-      <TimelineStep label="Requested" subLabel="3rd May 14:04" type="success">
-        We’ve assigned your request to one of our agents.
-      </TimelineStep>
-      <TimelineStep label="In progress" subLabel="4th May 10:25" type="success">
-        We’ve applied for a refund from the carrier(s).
-      </TimelineStep>
-      <TimelineStep label="Waiting for the carrier" subLabel="5th May 15:03" type="success">
-        We’ve applied for a refund from the carrier(s).
-      </TimelineStep>
-      <TimelineStep label="Carrier is refunding" subLabel="6th May 20:50" type="success">
-        The carrier(s) is processing your refund request. We’ll contact them again if necessary.
-      </TimelineStep>
-      <TimelineStep label={label} subLabel={time} type={type}>
-        {children}
-      </TimelineStep>
-    </Timeline>
-  );
-};
 
-WithCritical.story = {
-  name: "with critical",
-  parameters: { info: "This is example of Timeline with critical type" },
-};
-
-WithCritical.args = {
-  label: "Non refundable",
-  time: "7th May 10:30",
-  type: Types.Critical,
-  children: "The carrier rejected the refund",
-};
+  args: {
+    children: "We’ve applied for a refund from the carrier(s).",
+    label: "Waiting for the carrier",
```

Comment:
I would add `{ table: { category: xxx } }` to distinguish args for `Timeline` and `TimelineStep` components. I know that we were discussing this during the process on the last batched, we can include it to the `final fixes` ticket.